src/components/menu/menu.css

```diff
@@ -21,8 +21,8 @@
 
 .menu-close-button {
     position: absolute;
-    top: 40px;
-    left: 40px;
+    top: 20px;
+    left: 20px;
     cursor: pointer;
     z-index: 1001;
```

Comment:
I've just started think about "code of conduct" or a Confluence page or so... just have you to get use to it. What do you think?
This comment is not a call to action. It's a proposal. What do you think?

Reply:
That sounds like a great idea! I completely agree, and I think it would be awesome to have something like that in place.